templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/terraform/main.tf

```diff
@@ -73,3 +73,15 @@ data "azurerm_public_ip" "app_gateway_ip" {
   name                = "pip-agw-${var.tre_id}"
   resource_group_name = data.azurerm_resource_group.core.name
 }
+
+data "azurerm_key_vault_key" "ws_encryption_key" {
```

Comment:
Maybe worth adding it in a data.tf file like it is everywhere else?

Reply:
Done - created a data.tf file and moved everything